### DISCUSSION TITLE
Making the 'merge status list' items accessible to people using assistive technology

### DISCUSSION THREAD

**Discussion by peteryates** [community/community#28483] [Accessibility] [answered]
In the 'checks' list that appears at the bottom of pull requests, the 'Details' link on the right of each entry doesn't provide any context to screen reader users.

Sighted users get their context from the text content to the left, but screen reader users going from landmark to landmark may hear the links out of context and not know what they do. They could just hear 'Details' 10, 20 or 30 times in a row.

![Screenshot from 2022-08-05 16-59-02](https://user-images.githubusercontent.com/128088/183115892-9b086c7c-3581-4385-a0de-8979d5bc1e1b.png)

In the GOV.UK design system we [add some visually hidden text](https://design-system.service.gov.uk/components/summary-list/#adding-actions-to-each-row) to the link to provide the missing context.

Using the above example the links could be:

* Details (about the Semgrep scan)
* Details (on the test run with Ruby 2.7.6 and Rails 6.1.5)
* Details (on the test run with Ruby 2.7.6 and Rails 7.0.3)
* etc

**Comment by benjiallen** [comment]
Thanks for reporting this issue @peteryates. I agree the accessible name needs improvement. I'll get this feedback over to the right product manager.

**Comment by cehfisher** [comment, marked as answer]
Hello @peteryates - this issue seems to have been resolved with recent updates. Could you please confirm and close this issue? Thanks! (cc @queenofcorgis)

**Comment by peteryates** [comment]
Thank you @cehfisher.

I see there's now an `aria-label`, so the situation is definitely improved. Thank you.
